spec/requests/api/api_provides_client_with_movies_to_display_spec.rb

```diff
@@ -28,7 +27,7 @@
 
     it 'is expected to have movie poster in \'movies\'' do
       expect(response_json['movies'].third['poster_path'])
-      .to eq "/sp4zXS3x4wHyL8wm8zLioiBrxuR.jpg"
+      .to eq "/api/movies/image/sp4zXS3x4wHyL8wm8zLioiBrxuR.jpg"
```

Comment:
Why do you need to do this?

Reply:
It's because we thought it is better to have the complete link of image for testing but as I see it now it is just a test and i can test it against the json file. So I shouldn't have added that `/api/movies/images`